src/db-connection.ts

```diff
@@ -41,7 +41,13 @@ async function runMigrations(connection: Knex) {
 
 export async function createDbConnection(dbConfig: Db = config.db): Promise<Knex> {
   const knexConfig: Knex.Config = {
-    ...dbConfig,
+    client: dbConfig.client,
+    connection: dbConfig.connection.connectionString
```

Comment:
```suggestion
    connection: dbConfig.connection.connectionString || dbConfig.connection
```

If I am not mistaken, this achieves the same result, yet it is less complicated to read.